applications/FSIapplication/custom_utilities/nodal_update_utilities.h

```diff
@@ -107,33 +103,27 @@ class NodalUpdateBaseClass
      * @param timeStep: time step value
      */
     virtual void UpdateMeshTimeDerivatives(ModelPart& rModelPart,
-                                           const double timeStep)
-    {
+                                           const double timeStep) {
         KRATOS_ERROR << "Calling the nodal update base class UpdateMeshTimeDerivatives() method. Call the proper time scheme derived one.";
     }
 
     /**
      * Sets the fluid interface time derivatives as the mesh displacement computed values.
      * @param rInterfaceModelPart: modelpart in where the nodal update is to be performed
      */
-    virtual void SetMeshTimeDerivativesOnInterface(ModelPart& rInterfaceModelPart)
-    {
+    virtual void SetMeshTimeDerivativesOnInterface(ModelPart& rInterfaceModelPart) {
         auto& rLocalMesh = rInterfaceModelPart.GetCommunicator().LocalMesh();
         ModelPart::NodeIterator local_mesh_nodes_begin = rLocalMesh.NodesBegin();
         #pragma omp parallel for firstprivate(local_mesh_nodes_begin)
         for(int k=0; k<static_cast<int>(rLocalMesh.NumberOfNodes()); ++k)
         {
             ModelPart::NodeIterator it_node = local_mesh_nodes_begin+k;
 
-            array_1d<double, 3>& v_node = it_node->FastGetSolutionStepValue(VELOCITY);      // Current step interface velocity
-            array_1d<double, 3>& a_node = it_node->FastGetSolutionStepValue(ACCELERATION);  // Current step interface acceleration
-
-            noalias(v_node) = it_node->FastGetSolutionStepValue(MESH_VELOCITY);        // Set the current interface velocity as the mesh velocity;
-            noalias(a_node) = it_node->FastGetSolutionStepValue(MESH_ACCELERATION);    // Set the current interface acceleration as the mesh acceleration;
+            array_1d<double, 3>& v_node = it_node->FastGetSolutionStepValue(VELOCITY);  // Current step interface velocity
```

Comment:
just out of curiousity, isn't this implemented in the `VariableUtils`?

Reply:
In the past it was impossible to use the VariableUtils since we updated both de VELOCITY and the ACCELERATION. To update the acceleration has no implications so we remove this. At this point, probably you are right and we can use the VariableUtils(). I will have a look ;).